.goreleaser.yml

```diff
@@ -0,0 +1,126 @@
+# This is an example .goreleaser.yml file with some sane defaults.
+# Make sure to check the documentation at http://goreleaser.com
+project_name: autocert
+
+before:
+  hooks:
+    # You may remove this if you don't use go modules.
+    - go mod download
+    # - go generate ./...
+
+builds:
+  -
+    id: default
+    env:
+      - CGO_ENABLED=0
+    targets:
+      - linux_amd64
+    flags:
+      - -trimpath
+    main: ./controller
+    binary: autocert
+    ldflags:
+      - -w -X main.Version={{.Version}} -X main.BuildTime={{.Date}}
+
+archives:
+  -
+    # Can be used to change the archive formats for specific GOOSs.
+    # Most common use case is to archive as zip on Windows.
+    # Default is empty.
+    name_template: "{{ .ProjectName }}_{{ .Os }}_{{ .Version }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}_{{ .Mips }}{{ end }}"
+    format_overrides:
+      - goos: windows
+        format: zip
+    builds:
+      - default
+    wrap_in_directory: "{{ .ProjectName }}_{{ .Version }}"
+    files:
+      - README.md
+      - LICENSE
+
+source:
+  enabled: true
+  name_template: '{{ .ProjectName }}_{{ .Version }}'
```

Comment:
What is the reason behind this, GitHub does this by default

Reply:
The reason is Github could change what they do by default and we would still keep things consistent.